per/admin.py

```diff
@@ -155,6 +155,26 @@ class FormComponentQuestionAndAnswerAdmin(admin.ModelAdmin):
     pass
 
 
+class OpsLearningAdmin(admin.ModelAdmin):
+
+    def get_fields(self, request, obj=None):
```

Comment:
The problem with this approach is that the user will have to select `is_validated` then Save, then reload the form to see the `_validated` fields - I think the desired user behaviour here would be to immediately show the fields when the user selects the check-box. That would require some hacky javascript to do - let's clarify the desired behaviour from @ypyelab exactly and decide what to do here.

Reply:
Based on chat with @ypyelab in Teams:

Let's show all the fields all the time and just make the non- `_validated` fields not editable in the admin. The way to do this would be to add the fields that we don't want editable in the Admin to a `readonly_fields` array in the Admin definition like here: https://stackoverflow.com/a/3967891 and I think that should be all we need.